src/server/handlers.rs

```diff
@@ -41,6 +45,15 @@ impl<S: Store + Send + Sync> Handler<S> {
             "Manifest is valid. Fetching current manifests from store"
         );
 
+        if manifest.version() == LATEST_VERSION {
+            self.send_error(
+                msg.reply,
+                format!("A manifest with a version {LATEST_VERSION} is not allowed in wadm"),
```

Comment:
Make sure to document this bad boy 😉

Reply:
No changes needed here. Just something we'll want to note when mentioning OAM versions